graphbolt/src/csc_sampling_graph.cc

```diff
@@ -123,11 +123,46 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::InSubgraph(
 
 c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighbors(
     const torch::Tensor& nodes) const {
-  // TODO(#5692): implement this.
+  const int64_t num_nodes = nodes.size(0);
+
+  std::vector<torch::Tensor> picked_neighbors_per_node(num_nodes);
```

Comment:
chatgpt suggest to use `picked_neighbors` and `picked_neighbor_counts` for these two variables...